src/datapump/clients/data_api.py

```diff
@@ -52,6 +52,8 @@ def get_1x1_asset(self, dataset: str, version: str) -> str:
             )
         elif dataset == "gadm" and version == "v3.6":
             return "s3://gfw-files/2018_update/tsv/gadm36_adm2_1_1.csv"
+        elif dataset == "gadm" and version == "v4.1":
```

Comment:
Is it here because you needed to make manual changes to the 1x1 output from the API?